k8s-deploy/k8s/axon/axon1-statefulset.yaml

```diff
@@ -101,6 +101,15 @@ spec:
           resources: {}
           securityContext:
             privileged: true
+      affinity:
```

Comment:
can we use `nodeSelector`? this is cumbersome.

https://github.com/axonweb3/axon-devops/blob/0ea7a6dcfcb33b8d47b0d71d24079f56d58ab186/k8s-deploy/k8s/axon/axon1-statefulset.yaml#L96